### PR TITLE
change travis to github actions in testing documentation

### PR DESCRIPTION
**Pull request by individual-it**
travis seems not to be used anymore and CI runs in github actions

[skip ci]

### CODE REVIEW

File: docs/development/running-tests/README.md [hunk 1]
```diff
@@ -2,35 +2,42 @@
...
 
 
 
-Here you'll see that the *Travis CI* check has reported an error, which likely means that your pull request contains errors. It might also result from a temporary error running the test suite, or from a test that was broken in the `dev` branch.
+Here you'll see that the *Github Actions* check has reported an error, which likely means that your pull request contains errors. It might also result from a temporary error running the test suite, or from a test that was broken in the `dev` branch.
 
-If you expand the view  by clicking on details, you will see the individual *jobs* that Travis executes. The test suite is run in parallel to save time.  The overall run time of the test suite is around *3 - 4 hours* on Travis, but with parallel test execution, this time is reduced to around 30 - 40 minutes waiting time.
+If you expand the view  by clicking on details, you will see the individual *jobs* that Github executes. The test suite is run in parallel to save time.  The overall run time of the test suite is around *3 - 4 hours* on Github, but with parallel test execution, this time is reduced to around 30 - 40 minutes waiting time.
```
Comment (phil-davis) [inline docs/development/running-tests/README.md]: May as well make the global replace of Travis=>GitHub. The time estimates will later need to be adjusted to whatever is the usual approximate run time in GitHub actions.

**Review by phil-davis — APPROVED** [review]
LGTM

**Comment by oliverguenther** [comment]
Thanks for looking into this. It is very appreciated :bow:  I updated the section about run times to match the run times on GitHub.

File: docs/development/running-tests/README.md (hunk 1 above)
Comment (phil-davis) [reply docs/development/running-tests/README.md]: Done

**Pull request merged**